src/main/java/iroha/validation/transactions/provider/impl/MQTransactionProvider.java

```diff
@@ -1,45 +0,0 @@
-package iroha.validation.transactions.provider.impl;
-
-import io.reactivex.Observable;
-import iroha.protocol.BlockOuterClass.Block;
-import iroha.protocol.TransactionOuterClass.Transaction;
-import iroha.validation.transactions.provider.TransactionProvider;
-import org.springframework.stereotype.Component;
-import sun.reflect.generics.reflectiveObjects.NotImplementedException;
-
-/**
- * {@link TransactionProvider} implementation on message queues
- */
-@Component
-public class MQTransactionProvider implements TransactionProvider {
```

Comment:
Are we not going to have it implemented? 😝

Reply:
I think I will add it to iroha helper, since I don't see any use of MQ as Transaction provider :D